### PR TITLE
feat(postman): add folder for credential provider

### PR DESCRIPTION
**Pull request by Mingyang-Li-MATTR**
# ✍️ Description
Added new endpoints following api-ref v2 update, which includes all new endpoints for `Credential Provider`, but it doesn't include the updated endpoints for CRUD operations on `Web Semantic Credentials`. We'll be addressing them in another PR

## 📝 Context
Organised requests by their category as shown on API doc, except for a few differences. API doc currently doesn't have all the CRUD endpoints for `Credential Providers`, but this PR will add them as part of the collection

### CODE REVIEW

**Comment by Mingyang-Li-MATTR** [comment]
Making more changes before review. 

Just found out `Credential Configuration` sits under `Web Semantic Credentials`, not `Credential Provider` category. Making some adjustments atm.

**Comment by Mingyang-Li-MATTR** [comment]
Note during co-review:`GET TENANT_URL/core/v1/oauth/authorize` can be fixed later

File: postman/Platform Core API.postman_collection.json [hunk 1]
```diff
@@ -2665,6 +2761,554 @@
...
+              },
+              "response": []
+            }
+          ]
+        },
+        {
+          "name": "User",
+          "item": [
+            {
+              "name": "Search Users",
```
Comment (preetpatel) [inline postman/Platform Core API.postman_collection.json:3065]: should this be "Create user" ?

**Review by preetpatel — CHANGES_REQUESTED** [review]

File: postman/Platform Core API.postman_collection.json (hunk 1 above)
Comment (Mingyang-Li-MATTR) [reply postman/Platform Core API.postman_collection.json:3065]: I don't think it should.

There's no `create-user` endpoint on api-ref. Plus, we don't want devs to manually add `users` onto their tenant, they should've been synced by us every time a new user authenticates via oid4vci.

**Review by preetpatel — APPROVED** [review]

**Pull request merged**